src/thor/dijkstras.cc

```diff
@@ -181,11 +181,14 @@ void Dijkstras::ExpandInner(baldr::GraphReader& graphreader,
     // Check if the edge is allowed or if a restriction occurs
     EdgeStatus* todo = nullptr;
     uint8_t restriction_idx = -1;
+    // is_dest is false, because it is a traversal algorithm in this context, not a path search
+    // algorithm. In other words, destination edges are not defined for this Dijkstra's algorithm.
+    bool is_dest = false;
```

Comment:
```suggestion
    const bool is_dest = false;
```
If it's always false?